resources/views/header/header.blade.php

```diff
@@ -48,8 +48,8 @@
                                 </button>
                             </a>
                         </div>
-                        <div class="col-xs-4" style="margin:0; padding-top:3%">
-                                <a class="edit-user-name" href="/user/{{ Auth::user()->id }}/edit"><i class="fa fa-pencil" aria-hidden="true"></i></a>
+                        <div class="col-xs-2" style="margin-left:-20px; padding-top:3%">
```

Comment:
please move all styles in independent style file

Reply:
I don't know the better way to overwrite bootstrap styles rather then write style inside html)))